perfkitbenchmarker/network.py

```diff
@@ -22,20 +22,38 @@
 
 import threading
 
+from perfkitbenchmarker import context
+from perfkitbenchmarker import errors
+
 
 class BaseFirewall(object):
   """An object representing the Base Firewall."""
 
   # Lock used to serialize the instantiation of new BaseFirewall objects.
   _class_lock = threading.Lock()
```

Comment:
If these class locks are only used for the dictionaries in the benchmark spec, it would be nice to move them out of this class.